src/app/component/login/login.component.ts

```diff
@@ -50,6 +50,15 @@ export class LoginComponent implements OnInit {
   };
   // public data: LoginData;
 
+  readonly LANGUAGE_MAPPING = {
```

Comment:
We can't hard code value.
For example, if we need to add new languages, if we work in this way, we need to change code, build again new version, and put it on the server. We don't want it. All need to work via configuration form like before.
Also, if someone changes the language label via form this doesn't work after that.

Reply:
Done, please check 57c9397.